Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -1299,6 +1304,17 @@ public class PBXProjGenerator {
             targetObject.productType = target.type
         }
     }
+    
+    private func makePlatfor(for platform: Dependency.Platform) -> String? {
```

Comment:
Platfor

Reply:
Updated in dfc471d8511f37738bfb9a84915d822bcc299f1e.